guides/common/modules/ref_example-playbooks-based-on-modules-from-project-ansible-collection.adoc

```diff
@@ -0,0 +1,153 @@
+[id="example-playbooks-based-on-modules-from-{project-context}-ansible-collection"]
+= Example playbooks based on modules from {Project} Ansible Collection
+
+All playbooks based on modules from {Project} Ansible Collection must include parameters detailing how to connect to the {Project} API.
+The following examples use Ansible vault and module defaults group to provide these parameters, and they authenticate using a username and password.
+For more information, see xref:creating-a-playbook-with-modules-from-{project-context}-ansible-collection[].
+
+.Example Ansible playbook: Enable repositories and create a content view
+====
+This example playbook uses the following modules:
+
+* `{ansible-namespace}.repository_set`
+* `{ansible-namespace}.content_view`
+
+The playbook ensures repositories are enabled and a content view that contains these repositories exists.
+
+[options="nowrap" subs="+quotes,attributes"]
+----
+- name: Ensure RHEL 10 repositories are enabled and a content view exists
+  hosts: localhost
+  vars_files:
+    - _My_Vault.yml_
+  module_defaults:
+    group/{ansible-module-defaults-group}:
+      username: "{{ _My_Username_ }}"
+      password: "{{ _My_Password_ }}"
+      server_url: "{{ _My_Server_URL_ }}"
+  tasks:
+    - name: Ensure RHEL 10 repositories are enabled
+      {ansible-namespace}.repository_set:
+        name: "Red Hat Enterprise Linux 10 for x86_64 - BaseOS (RPMs)"
+        organization: "Default Organization"
+        product: "Red Hat Enterprise Linux for x86_64"
+        repositories:
+        - releasever: "10"
+          basearch: "x86_64"
```

Comment:
Since RHEL8 the `basearch` is not needed (and actually breaks things sometimes), as the `x86_64` is already in the product/repo name.

Reply:
Good to know, and `basearch` removed.

Reply:
I wonder if it means the examples in https://github.com/theforeman/foreman-ansible-modules/blob/develop/plugins/modules/repository_set.py should be updated. They are for RHEL 7 but still. Perhaps it might be a good idea to use a more recent version as an example?

Reply:
There are rhel8 examples if you scroll down, but indeed they could be more prominent